README.md

```diff
@@ -34,12 +34,27 @@ There are 2 analyses using Pseudomonas dataset in the `Pseudomonas` directory an
 
 ## How to run notebooks from simulate-expression-compendia
 
+**Operating Systems:** Mac OS, Linux
+
 In order to run this simulation on your own gene expression data the following steps should be performed:
 
 First you need to set up your local repository: 
-1. Clone the `simulate-expression-compendia` repository
-2. Install [miniconda](https://docs.conda.io/en/latest/miniconda.html)
-3. Set up conda environment by running the following command in the terminal:
+1. Clone the `simulate-expression-compendia` repository by running the following command in the terminal:
+```
+git clone https://github.com/greenelab/simulate-expression-compendia.git
+```
+2. Download and install [git extension](https://brew.sh/) for large files (git-lfs tracked files)
```

Comment:
Do you think it would be good to offload the instructions for git lfs to github's instructions? Brew is probably only going to work for OS X

Reply:
Ah..yes this makes sense, i'll update this